core/internal/cltest/factories.go

```diff
@@ -842,7 +842,7 @@ func MustInsertKeeperRegistry(t *testing.T, store *strpkg.Store, ethKeyStore *ke
 	registry := keeper.Registry{
 		ContractAddress:   contractAddress,
 		BlockCountPerTurn: 20,
-		CheckGas:          10_000,
+		CheckGas:          150_000,
```

Comment:
We're going to need a better way of configuring gas limits I think, since this starts to not work well on other chains. Maybe not necessary in this PR but we will need it soon.

Currently its inconsistent and scattered around the codebase. Would be nice to have one file where all the gas limits lived, that could return different values for different chains.

Maybe something like `chains/gas_limits.go` ?

Reply:
We have `KEEPER_REGISTRY_GAS_OVERHEAD`. Could that configuration live with the rest of config in `chains/configs.go`?